### ISSUE TITLE
Project Sign-up Form: Clarify lack of requirement for responsiveness

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28478]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the assignment, section 3 step 6, it says:

> Do not worry about making your project look nice on mobile, but DO resize your browser a little bit to make sure that it’s not completely broken at different desktop resolutions.

While this is intended to take the focus off responsive design but not hardfix specific values to the point where the slightest browser size change completely destroys the site, it's been a *very* common pitfall for many learners where they read too much into it and end up just trying to implement a fully responsive design. Clearly something about this wording can be made more explicit.

Ideally, we just state not to worry about responsive design because it's covered later, and that's it. No need to say much more, and trust learners to use the tools they've been learning about to achieve something reasonable.

Wording suggestion:

> Do not worry about making your project look nice on mobile. Responsive design isn't covered until later in the curriculum.

## Acceptance criteria

- [ ] Assignment section 3 step 6 is reworded to the above, or wording to that effect.

Comment below if you wish to be assigned. Please do not open a PR unless you have been assigned by a maintainer.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-sign-up-form

**Comment by mathdebate09** [comment]
Hey Mao, I can fix this issue up :D

**Comment by MaoShizhong** [comment]
All yours
